### ISSUE TITLE
Master Change List

### ISSUE THREAD

**Issue by cloaked2222** [S5SS/dual-command#64]
In addition to each build change log, a master change list noting all the changes since 2.3 build 9 is badly needed. Its quite hard to follow all the changes if you read them build by build. Some changes have be un-done/further changed, and its a lot easier to look at one race's changes all at once than piecemeal across 10+ builds. Additionally the detailed reasoning for all the changes in each build change log is great, but the master change list should probably be brief since there's so many changes.

Ideally put the master change list in a similar format to whats been used for all the other HWR patches:
https://forums.gearboxsoftware.com/t/homeworld-remastered-patch-balance-changes/1501771/1

The stats tool can assist here:
https://forums.gearboxsoftware.com/t/tool-stats-tool/1501938

Also if your not already using it, check out Meld. For 2.3 I would do a file by file comparison of each build to ensure the build change list was complete, and to try and spot mistakes like changing one races stat and forgeting the others.
https://meldmerge.org/

**Comment by Novaras** [comment]
[b10_from_2.3.zip](https://github.com/S5SS/dual-command/files/3164735/b10_from_2.3.zip)

tyguy did this, however its just a compilation of logs instead of an actual comparison between files

**Comment by cloaked2222** [comment]
Thanks tyguy! That's helpful.
